packages/algorand-js/test/mocks/stateful.ts

```diff
@@ -1,76 +1,58 @@
-import { AppLocalState } from "algosdk";
+import { AssetDef, AssetHolding } from "algosdk";
 
 import { stringToBytes } from "../../src/lib/parsing";
+import { AppLocalStateM, SSCAttributesM, StackElem } from "../../src/types";
+
+const byteToStr = (str: string): string => { return stringToBytes(str).toString(); };
+
+const appLocalState = new Map<number, AppLocalStateM>();
+const createdApps = new Map<number, SSCAttributesM>();
+const createdAssets = new Map<number, AssetDef>();
+
+const assets = new Map<number, AssetHolding>();
+assets.set(3, { 'asset-id': 3, amount: 2, creator: "string", 'is-frozen': "false" });
+assets.set(32, { 'asset-id': 32, amount: 2, creator: "AS", 'is-frozen': "false" });
+
+const globalKeyVal = new Map<string, StackElem>();
+globalKeyVal.set(byteToStr('Hello'), stringToBytes('World'));
```

Comment:
can we rename key-val in global as well ?

Reply:
I don't follow. What should we rename here?

Reply:
maybe globalstatemap

Reply:
works